Tools/peg_generator/scripts/benchmark.py

```diff
@@ -90,11 +90,8 @@ def run_benchmark_stdlib(subcommand, parser):
     for _ in range(3):
```

Comment:
The `--parser` argument for this script is confusing because we have `pegen` and `cpython`. I think this should be `new` vs `old` to avoid confusion.